src/Twig/GlobalVariables.php

```diff
@@ -87,6 +98,11 @@ public function getAdminPool()
      */
     public function url($code, $action, $parameters = [], $referenceType = UrlGeneratorInterface::ABSOLUTE_PATH)
     {
+        @trigger_error(sprintf(
```

Comment:
Is there any replacement method for this method @VincentLanglet ?

Our other bundles are still using this method, e.g. https://github.com/sonata-project/SonataPageBundle/blob/6b80c4a675303b2ad707655150dc02fd0be43efc/src/Resources/views/Block/block_pagelist.html.twig#L26

Reply:
I would recommend to pass the PageAdmin or the AdminPool in the BlockService https://github.com/sonata-project/SonataPageBundle/blob/3.x/src/Block/PageListBlockService.php

And then calling `pageAdmin.generateUrl()`